src/app/StoreProvider.tsx

```diff
@@ -192,6 +230,29 @@ const produceWithUndo = (previous: AccessibleForm, producer: Producer) => {
   });
 };
 
+// See https://github.com/allenai/pawls/blob/3cc57533248e7ca787b71cafcca5fb66e96b2166/ui/src/context/PDFStore.ts#L31
+// FIXME: This logic should be moved out of store file.
```

Comment:
Good point, adding a `utils` file makes a lot of sense.

Reply:
Done.